### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by 4D42**
Add instruction for suite2p, fix broken link for CaImAn, small other fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,16 +21,33 @@ pyinstaller pack_minian_run.spec
...
 ```
 pyinstaller pack_caiman_run.spec
 ```
+
+## Suite2p
+---
+
+[Install Suite2p](https://suite2p.readthedocs.io/en/latest/installation.html) and Pyinstaller in the same anaconda environment.
+
+Start a command prompt in Caiman environment and go to the git directory.
```
Comment (doriclenses) [inline README.md:43]: Caiman?

File: README.md [hunk 2]
```diff
@@ -21,16 +21,33 @@ pyinstaller pack_minian_run.spec
 ## Caiman
```
Comment (doriclenses) [inline README.md:21]: CaImAn

**Review by doriclenses — APPROVED** [review]

**Comment by doriclenses** [comment]
Any reason it's a draft?

**Comment by doriclenses** [comment]
@4D42 should we merge

**Review by kapilseh — APPROVED** [review]

**Pull request merged**